### PR TITLE
Aws ci tf main

### PR DESCRIPTION
**Pull request by spisarski**
#### What does this PR do?
Removed top-level ansible script and now CI will be called directly from TF
Removed ssh keys and now the host must send in the pub/priv keys
Locked down VM ports
#### Do you have any concerns with this PR?
no
#### How can the reviewer verify this PR?
run CI
#### Questions:
- Have you connected this PR to the issue it resolves?
n/a
- Does the documentation need an update?
yes, we will update once we have integrated into a CI server
- Does this add new Python dependencies?
no
- Have you added unit or functional tests for this PR?
no
- Does this patch update any configuration files?
no

### CODE REVIEW

**Comment by bo-quan** [comment]
Can one of the admins verify this patch?

File: ci/aws/provisioner.tf [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+# limitations under the License.
+
+# Call ensure SSH key has correct permissions
+resource "null_resource" "snaps-k8s-setup" {
+  provisioner "local-exec" {
+    command = "chmod 600 ${var.private_key_file}"
+  }
+}
+
+# Call ansible scripts to setup OVS
```
[the remark this answers is not in the document]
Comment (spisarski) [reply ci/aws/provisioner.tf]: Ugh, I forgot a lot here. Fixed if you can re-review

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Comment by bo-quan** [comment]
Run Tests

**Pull request merged**